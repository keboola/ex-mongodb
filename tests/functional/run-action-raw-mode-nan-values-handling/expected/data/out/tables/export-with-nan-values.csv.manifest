{"primary_key":["id"],"incremental":false}
