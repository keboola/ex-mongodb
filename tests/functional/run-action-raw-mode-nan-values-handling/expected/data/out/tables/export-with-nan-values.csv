"id","data"
"5716054bee6e764c94fa8181","{""_id"":{""$oid"":""5716054bee6e764c94fa8181""},""key"":1}"
"5716054bee6e764c94fa8182","{""_id"":{""$oid"":""5716054bee6e764c94fa8182""},""key"":{""$numberDouble"":""NaN""}}"
"5716054bee6e764c94fa8183","{""_id"":{""$oid"":""5716054bee6e764c94fa8183""},""key"":3}"
"5716054bee6e764c94fa8184","{""_id"":{""$oid"":""5716054bee6e764c94fa8184""},""key"":4}"
"5716054bee6e764c94fa8185","{""_id"":{""$oid"":""5716054bee6e764c94fa8185""},""key"":{""$numberDouble"":""NaN""}}"
"5716054bee6e764c94fa8186","{""_id"":{""$oid"":""5716054bee6e764c94fa8186""},""key"":6}"
